components/css/newsletter-text-only/_contentPage.scss

```diff
@@ -0,0 +1,195 @@
+$white:#ffffff;
+$black:#000000;
+$light-gray:#aaaaaa;
+$dark-gray:#2e2e2e;
+$campusGold:#c28e0e;
+
+.intro {
+	background-color: $campusGold;
+	padding: 1.5em;
+	
+		&__title {	
+			font-size:calc-toRem(32);
+			line-height: 1.3em;
+			color: $black;
+			text-align: center;
+			text-transform:uppercase;
+			padding: .5em 0;
+		}
+	
+		&__issue {
+			font-size: 1em;
+			line-height: 1em;
+			color: $black;
+			text-align: center;
+			padding: 0;
+		}
+	
+		&__issue::after {
+			content:" ";
+			display: block;
+			width: 30%;
+			height: 3px;
+			margin: .3em auto !important;
+			background-color: $white;	
+		}
+	
+		&__description {
+			font-size: .9em;
+			line-height: 1.4em;
+			color: $black;
+			text-align: center;
+			padding-top: 1.5em;
+		}
+	}
+
+
+.section {
+	background-color: $dark-gray;
+	padding: 1.5em;
+	
+		&__heading {
+			font-family:$font-champion-middle;
+			color: $white;
+			font-size: 1.5em;
+			line-height: 1.2em;
+			text-transform:uppercase;
+			padding-top: .5em;
+				&-light {
+					color: #000000;
+					}
+			}
+		
+		&__heading::after {
+			content:" ";
+			display: block;
+			width: 25%;
+			height: 3px;
+			background-color: $campusGold;
+			margin: .2em 0 0 0 !important;
+			}
+	
+
+		  &__list{
+			  &-other{
+				padding:0;
+				list-style-type:none;
+			  }
+		  }
+  
+		&__item {
+			&-other {
+				display:block;
+				overflow:hidden;
+			 	padding:20px 0 0;
+					  &:first-child {
+					  padding:0;
+					  }
+			}
+  		}
+	}
+
+.story {
+	margin: 0;
+	padding: 0;
+	
+		&__title {
+			color: $campusGold;
+			font-size: 1.4em;
+			line-height: 1.3em;
+			padding: 1em 0 0 0;
+				&-light {
+					color: #000000;
+				}
+		}
+	
+		&__subtitle {
+			color: $light-gray;
+			font-size: 1em;
+			line-height: 1.5em;
+			padding-top: .5em;
+			padding-bottom: .8em;
+		}
+	
+		&__teaser {
+			color: $white;
+			font-size: .9em;
+			line-height: 1.5em;
+			padding-bottom: .5em;
+		}
+	
+		&--light {
+			background-color:$white;	
+		}
+	
+		&__link {
+			font-size: .9em;
+			line-height: 1.4em;
+				&-other {   
+					display:flex;
+					align-items:center;
+					margin-top: 20px;
+        
+				&:link,&:visited,&:hover,&:active{
+					color:$campusGold;
+					text-decoration:none;
+				}
+					
+				&::before{
+					content:"\f1ea";
+					font-family:FontAwesome;
+					display:flex;
```

Comment:
Garrett wants the icon to align with the top of the article if the title grows to multiple lines. If that makes sense... put another way, the text should flow down, but the position of the icon should stay the same in relationship to the first line of the title.

Reply:
So we don't need to use flex on the link?